.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# JIRA-123 One Sentence Overview
+
+**Work in progress. Don't merge!** State it here and use labels.
+
+## Links
+
+* http://wikia-inc.atlassian.net/browse/JIRA-123
+* more relevant links
+
+## Description
+
+A description written using complete sentences of the changes being made and any context necessary to understand them.
+
+## SQL Review
```

Comment:
that means that for EVERY PR I'll have to remove that
through my career at wikia I have made ONE PR with sql query in it.